### PR TITLE
feat(billing.autorenew): improve message when no service is excluded

### PR DESCRIPTION
**Pull request by marie-j**
##  improve message when no service is excluded

MANAGER-3282

### CODE REVIEW

File: client/app/billing/autoRenew/disable/disable.html [hunk 1]
```diff
@@ -1,5 +1,6 @@
 <div data-ui-view class="container-fluid">
     <billing-autorenew-bulk
+        data-complete-description="{{ 'billing_autorenew_disable_bulk' | translate }}"
```
Comment (FredericEspiau) [inline client/app/billing/autoRenew/disable/disable.html]:
```suggestion
        data-complete-description="{{:: 'billing_autorenew_disable_bulk' | translate }}"
```

File: client/app/billing/autoRenew/bulk/bulk.html [hunk 2]
```diff
@@ -16,12 +16,19 @@
                 count: $ctrl.servicesCount
             }">
         </strong>
+        <div data-ng-if="$ctrl.unavailableServicesCount === 0">
+            <p
```
Comment (FredericEspiau) [inline client/app/billing/autoRenew/bulk/bulk.html]:
```suggestion
            <p data-ng-if="$ctrl.unavailableServicesCount === 0"
```

**Comment by jleveugle** [comment]
Hello @marie-j,
Your pull request is now closed due to inactivity.
Feel free to re-open it if you have any update.
Anna Bot for OVH UX <3

File: client/app/billing/autoRenew/bulk/translations/Messages_fr_FR.json [hunk 3]
```diff
@@ -1,5 +1,6 @@
 {
   "billing_autorenew_bulk": "Vous avez sélectionné {{ count }} services",
+  "billing_autorenew_bulk_one": "Vous avez sélectionné 1 service",
```
Comment (FredericEspiau) [inline client/app/billing/autoRenew/bulk/translations/Messages_fr_FR.json]:
```suggestion
  "billing_autorenew_bulk_one": "Vous avez sélectionné un service",
```
In French you are supposed to write in letters numbers between 0 and 9 but this is nitpicking :)

File: client/app/billing/autoRenew/disable/translations/Messages_fr_FR.json [hunk 4]
```diff
@@ -5,6 +5,9 @@
   "billing_autorenew_disable_renew": "Il vous suffira de renouveler votre service si vous souhaitez le conserver.",
   "billing_autorenew_disable_data": "Attention, une fois le service expiré, il sera immédiatement suspendu, il ne sera plus accessible, et les données liées à ce service seront définitivement supprimées au bout de quelques jours sans règlement de votre part",
   "billing_autorenew_disable_bulk_unavailable": "Le renouvellement automatique sera désactivé pour l'ensemble de vos services sélectionnés, exceptés pour {{ count }} services :",
+  "billing_autorenew_disable_bulk_unavailable_one": "Le renouvellement automatique sera désactivé pour l'ensemble de vos services sélectionnés, exceptés pour 1 service :",
```
Comment (FredericEspiau) [inline client/app/billing/autoRenew/disable/translations/Messages_fr_FR.json]:
```suggestion
  "billing_autorenew_disable_bulk_unavailable_one": "Le renouvellement automatique sera désactivé pour l'ensemble de vos services sélectionnés, à l'exception de ce service",
```
?

File: client/app/billing/autoRenew/disable/translations/Messages_fr_FR.json [hunk 5]
```diff
@@ -5,6 +5,9 @@
...
+  "billing_autorenew_disable_bulk": "Le renouvellement automatique sera désactivé pour l'ensemble de vos services sélectionnés.",
```
Comment (FredericEspiau) [inline client/app/billing/autoRenew/disable/translations/Messages_fr_FR.json]:
```suggestion
  "billing_autorenew_disable_bulk": "Le renouvellement automatique sera désactivé pour tous les services sélectionnés.",
```

File: client/app/billing/autoRenew/enable/translations/Messages_fr_FR.json [hunk 6]
```diff
@@ -3,7 +3,10 @@
   "billing_autorenew_enable_payment_mean": "Une fois le renouvellement automatique mis en place pour vos services sélectionnés, le moyen de paiement utilisé sera votre moyen de paiement par défaut à savoir : {{ type }} {{ label }}",
   "billing_autorenew_enable_payment_mean_default": "Pour changer votre moyen de paiement par défaut, rendez-vous dans la section \"Mes moyens de paiement\"",
   "billing_autorenew_enable_payment_mean_frequency": "La fréquence de paiement sera configurée sur la fréquence minimum possible par service.",
-  "billing_autorenew_enable_bulk_unvailable": "Le renouvellement automatique sera activé pour l'ensemble de vos services sélectionnés, exceptés pour {{ count }} services :",
+  "billing_autorenew_enable_bulk_unavailable": "Le renouvellement automatique sera activé pour l'ensemble de vos services sélectionnés, exceptés pour {{ count }} services :",
```
Comment (FredericEspiau) [inline client/app/billing/autoRenew/enable/translations/Messages_fr_FR.json]:
```suggestion
  "billing_autorenew_enable_bulk_unavailable": "Le renouvellement automatique sera activé pour l'ensemble des services sélectionnés, sauf pour {{ count }} services :",
```

File: client/app/billing/autoRenew/enable/translations/Messages_fr_FR.json [hunk 7]
```diff
@@ -3,7 +3,10 @@
...
+  "billing_autorenew_enable_bulk_unavailable_one": "Le renouvellement automatique sera activé pour l'ensemble de vos services sélectionnés, exceptés pour 1 service :",
```
Comment (FredericEspiau) [inline client/app/billing/autoRenew/enable/translations/Messages_fr_FR.json]:
```suggestion
  "billing_autorenew_enable_bulk_unavailable_one": "Le renouvellement automatique sera activé pour l'ensemble des services sélectionnés, sauf pour ce service",
```

File: client/app/billing/autoRenew/enable/translations/Messages_fr_FR.json [hunk 8]
```diff
@@ -3,7 +3,10 @@
...
+  "billing_autorenew_enable_bulk": "Le renouvellement automatique sera activé pour l'ensemble de vos services sélectionnés",
```
Comment (FredericEspiau) [inline client/app/billing/autoRenew/enable/translations/Messages_fr_FR.json]:
```suggestion
  "billing_autorenew_enable_bulk": "Le renouvellement automatique sera activé pour l'ensemble des services sélectionnés",
```

File: client/app/billing/autoRenew/disable/translations/Messages_fr_FR.json (hunk 4 above)
Comment (marie-j) [reply client/app/billing/autoRenew/disable/translations/Messages_fr_FR.json]: won't match with the current display

Comment (FredericEspiau) [reply client/app/billing/autoRenew/disable/translations/Messages_fr_FR.json]: But isn't it better ? 😄

Comment (marie-j) [reply client/app/billing/autoRenew/disable/translations/Messages_fr_FR.json]: Not in the current context :) we do not only display the services but the reason why they're not eligible for activation/deactivation (actually we display the reason and the customer can expand the list of impacted services). So I'm not sure this content would make sense (given that context)

**Comment by antleblanc** [comment]
@marie-j Do you plan to address @FredericEspiau feedback regarding translations update?

**Comment by marie-j** [comment]
> @marie-j Do you plan to address @FredericEspiau feedback regarding translations update?

@antleblanc could do but this content has gone through Quality check as it is. some of the remarks are not relevant given the context imo

**Pull request merged**